kong-ingress/install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+KONG_VERSION=2.5
+KONG_INGRESS_VERSION=1.3.1
+NS=kong
+
+# Add the Helm repository for Kong Ingress Controller
+helm repo add kong https://charts.konghq.com
+helm repo update
+
+# Create the namespace and install the Helm chart
+kubectl create namespace $NS
+helm install kong kong/kong --set ingressController.installCRDs=false --set ingressController.image.tag=$KONG_INGRESS_VERSION --set image.tag=$KONG_VERSION --namespace $NS
```

Comment:
is installations of the CRD's is there in the quick start which mentioned in post install?

Reply:
Nope I don't see it. Now I see even the [default value for installCRD is set to false](https://github.com/Kong/charts/blob/main/charts/kong/README.md#ingress-controller-parameters).

Btw there are [different deployment options for kong ingress](https://github.com/Kong/charts/blob/main/charts/kong/README.md#deployment-options), and what I used is the default without tweaking anything.

Reply:
Can we have multiple deploy options just like we have for Kyverno or Linkerd @milindchawre ?

Reply:
I closely have a look, it seems the main difference between their deployment option is the [runtime option either open source or enterprise version of kong](https://github.com/Kong/charts/blob/main/charts/kong/README.md#runtime-package), which we can't opt since we need to stick to only open source.

Reply:
ok cool then will merge. Thanks for doing the research